lib/_tls_wrap.js

```diff
@@ -228,7 +228,7 @@ function initRead(tls, wrapped) {
  */
 
 function TLSSocket(socket, options) {
-  this._tlsOptions = options;
+  this._tlsOptions = options || { isServer: false };
```

Comment:
I don't think that this can possibly suffice. There are also context and various other things that needs to be specified. It can't be possible to omit them, so I don't think that there is any point in providing this default values.